### ISSUE TITLE
Investigate IcBadge Cypress visual and a11y tests - CI Build Failure

### ISSUE THREAD

**Issue by gd2910** [mi6/ic-ui-kit#1418]
## Summary of the bug 
The A11y Cypress tests for IcBadge are intermittently failing in the CI build, also visual tests are failing when ran locally.
The purpose of this ticket is to investigate the failures.
Fix may include increasing threshold 

## 🪜 How to reproduce 
A11y failures can be seen intermittently on PRs in the CI
Run the IcBadge Cypress tests locally to see visual failures

## 🧐 Expected behaviour 
The tests should consistently pass.

## 📝 Acceptance Criteria  

*Given* 
The CI pipeline is ran
*When*  
A PR is created
*Then* 
All tests should pass

## Additional info 
Visual tests are failing due to threshold, see the log in cypress testing for failure threshold.
Padding the tests or wait() may fix the A11y test
